src/app/accounts/index.types.d.ts

```diff
@@ -0,0 +1,10 @@
+type AddApiKeyForAccountArgs = {
+  accountId: AccountId
+  label?: string
```

Comment:
No optional arguments for use-cases please. Optional arguments on the use-case level are a smell that point to there actually being more than 1 use case in play.